biz.aQute.bndlib/src/aQute/bnd/osgi/repository/SimpleIndexer.java

```diff
@@ -141,4 +154,170 @@ public static void index(Collection<File> files, OutputStream outputStream, URI
 
 		repository.save(outputStream);
 	}
+
+	private static class SafeResourceBuilder extends ResourceBuilder {
```

Comment:
Since ResourceBuilder is a concrete class, we will add methods to it from time to time. So this class may be ignore and leak new methods. Perhaps it would be better to move this class to ResourceBuilder and add a `ResourceBuilder safeResourceBuilder()` method to ResourceBuilder which returns an object of this implementation. Then anyone changing ResourceBuilder will see this class and thus also change it?